crates/teloxide-core/src/types/chat_permissions.rs

```diff
@@ -51,36 +51,67 @@ bitflags::bitflags! {
         /// locations and venues.
         const SEND_MESSAGES = 1;
 
-        /// Set if the user is allowed to send audios, documents,
-        /// photos, videos, video notes and voice notes, implies
+        /// Set if the user is allowed to send audios. implies
+        /// `SEND_MESSAGES`.
+        const SEND_AUDIOS = (1 << 1) | Self::SEND_MESSAGES.bits;
```

Comment:
Can you keep the old values? I.e. such that `1 << 1` is unused, all already existing variants keep their tags, `SEND_AUDIO = (1 << 9)`, etc.

Reply:
And `SEND_MEDIA_MESSAGES`? make it last one?

Reply:
Not sure what you mean by the "last one", it is not supposed to have its own tag?